packages/ui/Button/index.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import clsx from 'clsx';
+
+import { Spinner } from '../Spinner';
+
+export interface ButtonProps
+  extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  variant?: 'primary' | 'secondary' | 'ghost';
+  isLoading?: boolean;
+}
+
+export const Button = ({
+  children,
+  variant = 'primary',
+  className,
+  isLoading,
+  ...props
+}: ButtonProps) => {
+  if (variant === 'ghost') {
+    return (
```

Comment:
I don't get what you are doing here, don't think anything related to the variant ghost is being added here?

Reply:
maybe ghost is not the right name, I just want the button to not have any styles.

Reply:
ah got it, lets not hold the PR for long. We can figure this out later